src/environmentd/Cargo.toml

```diff
@@ -115,17 +119,16 @@ datadriven = "0.6.0"
 fallible-iterator = "0.2.0"
 itertools = "0.10.5"
 jsonwebtoken = "8.2.0"
+mz-environmentd = { path = "../environmentd", features = ["test"] }
```

Comment:
Maybe this is where the default deps are pulled in?

Reply:
Sure, but this is in `dev-dependencies`. Does the restriction against jemalloc also apply to tests?

Reply:
I've added a modification to the `cargo tree` call to exclude `dev` dependencies. LGTY?

Reply:
Tested it, works as it's supposed to! Thank you.